fl4health/clients/nnunet_client.py

```diff
@@ -188,6 +189,9 @@ def __init__(
         # Used to redirect stdout to logger
         self.stream2debug = StreamToLogger(FLOWER_LOGGER, DEBUG)
 
+        # Used to scale gradients if using mized precision training (true if device is cuda)
```

Comment:
nit: mixed

Reply:
Should we add a specific argument here rather than making mixed precision the default for cuda. Is there ever any reason not to use mixed precision?